File: helloworld/javascript/level1/waterMelon.js
function waterMelon(n){
    var result = "";
    for (var i = 0; i < n; i++) {
        result += i%2 === 0 ? '수' : '박';
    }
    return result;
}

// 실행을 위한 테스트코드입니다.
console.log("n이 3인 경우: "+ waterMelon(3))
console.log("n이 4인 경우: "+ waterMelon(4))

File: helloworld/java/level1/WaterMelon.java
package level1;

public class WaterMelon {
	public String watermelon(int n) {
		StringBuffer sb = new StringBuffer();
		for (int i = 0; i < n/2; i++) {
			sb.append("수박");
		}
		if(n%2 == 1) {
			sb.append("수");
		}
		return sb.toString();
	}
	
	public static void main(String[] args) {
		WaterMelon wm = new WaterMelon();
		System.out.println("n이 3인 경우: " + wm.watermelon(3));
		System.out.println("n이 4인 경우: " + wm.watermelon(4));
	}
}
